client/my-sites/design-preview/index.js

```diff
@@ -220,20 +221,20 @@ function mapStateToProps( state ) {
 		};
 	}
 
+	const previewSlug = getPreviewSlug( state );
 	const { previewMarkup, customizations, isUnsaved } = state.preview[ selectedSiteId ];
 	return {
 		selectedSite,
 		selectedSiteId,
+		previewSlug,
 		previewMarkup,
 		customizations,
 		isUnsaved,
 	};
 }
 
 function mapDispatchToProps( dispatch ) {
-	return {
-		actions: bindActionCreators( PreviewActions, dispatch ),
-	};
+	return bindActionCreators( { fetchPreviewMarkup, undoCustomization, clearCustomizations, clearPreviewSlug }, dispatch );
```

Comment:
More convenient (and trivially more performant) is to pass `mapDispatchToProps` as an object instead of a function:

```js
export default connect(
	mapStateToProps,
	{ fetchPreviewMarkup, undoCustomization, clearCustomizations, clearPreviewSlug }
)( DesignPreview );
```